pkg/gofr/gofr.go

```diff
@@ -92,9 +95,53 @@ func New() *App {
 
 	app.subscriptionManager = newSubscriptionManager(app.container)
 
+	// static fileserver
+	currentWd, _ := os.Getwd()
+	checkDirectory := filepath.Join(currentWd, publicDir)
+
+	if _, err = os.Stat(checkDirectory); err == nil {
+		app.AddStaticFiles(publicDir, checkDirectory)
+	}
+
 	return app
 }
 
+func updateConfigInformation(config *gofrHTTP.StaticFileConfig, envConfig config.Config) {
+	config.DirectoryListing, _ = strconv.ParseBool(envConfig.GetOrDefault("STATIC_DIRECTORY_LISTING", "true"))
+	config.HideDotFiles, _ = strconv.ParseBool(envConfig.GetOrDefault("STATIC_HIDEDOTFILES", "true"))
+	config.ExcludeExtensions = strings.Split(envConfig.Get("STATIC_EXCLUDE_EXTENSIONS"), ",")
+	config.ExcludeFiles = strings.Split(envConfig.Get("STATIC_EXCLUDE_FILES"), ",")
```

Comment:
Here is your scapegoat @KedarisettiSreeVamsi ,you have to split only when the env variable is not empty

You should create a splitEnv (or whatever the name that
- will return an empty string when the variable name is empty
- split
- loop on them
- trim spaces on them

Reply:
done